### PR TITLE
chore: Add vite test lib

### PR DESCRIPTION
**Pull request by connoratrug**
- add lib dep
- configure test setup
- call via gradle

### CODE REVIEW

File: apps/molgenis-components/src/components/utils.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { assert, test } from "vitest";
+import { deepClone } from "./utils";
+
+// Edit an assertion and save to see HMR in action
```
Comment (chinook25) [inline apps/molgenis-components/src/components/utils.test.js:4]: what does this comment mean?

File: apps/molgenis-components/src/components/utils.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+test("deepClone", () => {
+  const input = {
+    foo: "hello",
+    bar: "world",
+  };
```
Comment (chinook25) [inline apps/molgenis-components/src/components/utils.test.js:10]: Might be good to have an object that goes a layer deeper for this test.

File: apps/molgenis-components/src/components/utils.test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { assert, test } from "vitest";
+import { deepClone } from "./utils";
+
+// Edit an assertion and save to see HMR in action
+
+test("deepClone", () => {
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/molgenis-components/src/components/utils.test.js:6]: good question , i dont think there is a convention provided by vitetest ( but would need to check this ) , could just add more tests i would assume

**Review by mswertz — APPROVED** [review]

File: apps/molgenis-components/src/components/utils.test.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (chinook25) [reply apps/molgenis-components/src/components/utils.test.js:6]: Ah it also has the describe: https://vuejs.org/guide/scaling-up/testing.html#unit-testing
So function name in describe(), flow explanation in test()

**Pull request merged**